internal/provider/features.go

```diff
@@ -25,6 +25,20 @@ func schemaFeatures(supportLegacyTestSuite bool) *pluginsdk.Schema {
 			},
 		},
 
+		"application_insights": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"disable_generated_rule": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+					},
+				},
+			},
+		},
```

Comment:
this needs to be added to the Features page, and also to the complete Example usage there (highlighting the default value of `false`): https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/features-block

Reply:
Fixed!